.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+# Description
+
+Summary of the change made.
+
+## Related Issues
+
+Tag the related issue.
+
+## Type of Change
+
+-[] Bug Fix (non-breaking change that fixes an issue)
+-[] New feature (non-breaking change which adds functionality)
+-[] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+# Checklist
+
+-[] The pull request is for branch `develop`
+-[] `black` gave 0 errors
+-[] `flake` gave 0 errors
```

Comment:
These could be [git hooks](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks)

Reply:
Yeah was about to do that only. Thought we could explicitly mention in the template anyways but I guess it's fine.